### ISSUE TITLE
column alignment problem

### ISSUE THREAD

**Issue by Zorospro** [dudaster/ele-custom-skin#292]
Hi! 
I have set the column number to 3 in the product archive but only 2 columns are displayed [https://sevabooks.eu/product-category/%d1%85%d1%83%d0%b4%d0%be%d0%b6%d0%b5%d1%81%d1%82%d0%b2%d0%b5%d0%bd%d0%b0-%d0%bb%d0%b8%d1%82%d0%b5%d1%80%d0%b0%d1%82%d1%83%d1%80%d0%b0/](url) 

![Екранна снимка (9)](https://user-images.githubusercontent.com/69197573/89286643-3ce17d00-d65b-11ea-9d3a-024642a695cb.png)

![Екранна снимка (10)](https://user-images.githubusercontent.com/69197573/89286647-3d7a1380-d65b-11ea-889a-6eb916494290.png)

How can I solve that issue? Also, I set the price weight to normal but it appears bold on the archive page. The discounted price of the first product is not displaying, only the regular one. I used the Astra theme but switched to Hello theme.

Thank you in advance.

**Comment by dudaster** [comment]
try remove the text widget and let me know if it is the same?

**Comment by Zorospro** [comment]
![Екранна снимка (11)](https://user-images.githubusercontent.com/69197573/89289925-b760cb80-d660-11ea-8f4f-6eec82a127cc.png)
The column width error disappeared but I am making a bookstore website and the text widget  is the author of the book 
(ACF used). Can you fix it? Also, the price problem remained - On the first product, only the normal price is shown.

**Comment by dudaster** [comment]
In a single template it works ok?

**Comment by Zorospro** [comment]
Yes, when I design the product loop it works great even with the text editor. The preview below the loop works great as well, but on the archive page it is broken.

**Comment by dudaster** [comment]
The issue with the price I meant.
